### ISSUE TITLE
console.log() not in order

### ISSUE THREAD

**Issue by kraftwer1** [Microsoft/vscode-chrome-debug#425]
- VS Code Version: 1.12.1

Steps to reproduce:

1. Add multiple console.log() accross a source file
2. Compare output of VS Code "Debug Console" and with that of Chrome Console

<img width="695" alt="bildschirmfoto 2017-05-07 um 15 24 49" src="https://cloud.githubusercontent.com/assets/2146436/25781415/482004ce-333a-11e7-97de-ef4a22991f83.png">
<img width="250" alt="bildschirmfoto 2017-05-07 um 15 24 57" src="https://cloud.githubusercontent.com/assets/2146436/25781416/48215c66-333a-11e7-94ce-cf9d723882e8.png">

First, I print out an array. Then I loop over it and print out the current iteration. Note that there's nothing asynchronous going on.

Might as well be a bug in Chrome itself as VS Code is probably just displaying what it gets forwarded.

**Comment by roblourens** [comment]
Can I see a code snippet that reproes this?

**Comment by kraftwer1** [comment]
I've tried to set it up again, but strangely, since restarting the toolings (Code and Chrome) it's not happening anymore...
The only thing I can say is that it happened on every reload, meaning it was completely reproducable at this time...

**Comment by shaunbaruth** [comment]
@roblourens @auchenberg I'm able to reproduce this. It seems to happen when rapidly logging a mix of plain text and objects. I'm able to reproduce every time. I tried restarting vscode as @kraftwer1 suggested, but it did not resolve the issue. I'm on vscode version 1.15.1. 

```
const o = {};
console.log('test');
console.log(o);
console.log('test2');
```

In the vscode debug console, you'll see:
test
test2
{}

Running via node from the command line will properly print out:
test
{}
test2

If I set debug breakpoints and step through slowly, the output is in the correct order (matches node output).
